indexed_gzip/tests/ctest_indexed_gzip.pyx

```diff
@@ -609,6 +609,23 @@ def test_iter(drop):
                  next(f)
 
 
+def test_index_points():
```

Comment:
Thanks for the unit test .. would you be able to extend it to a scenario where multiple seek points are generated?